app/benchmarks/results.md

```diff
@@ -0,0 +1,839 @@
+<!-- markdownlint-disable -->
```

Comment:
[question] why disable markdownlint for this file?

Reply:
because it complains about redundant headings all over the document so I decided not to lint it